client/src/state/anchorSelector/AnchorSelectorStates.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  SUCCESS: 'SUCCESS', // Done, or nothing selected
```

Comment:
how come `ActionTypes` has been renamed to `States`?
please keep to conventions already set, or bring up a discussion if you're going to change it...

Reply:
Just discussed this offline; States.js and ActionTypes.js are separate, not a mistaken synonyms. ;)